Assets/Scripts/MultiBlockCreator.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MultiBlockCreator : MonoBehaviour
+{
+    public List<float> staticBlockXPos = new List<float>()
+    {
+        2f, 4f, 6f, 8f, 10f, 12f, 14f,
+        2f, 4f, 6f, 8f, 10f, 12f, 14f,
+        2f, 4f, 6f, 8f, 10f, 12f, 14f,
+    };
+    
+    public List<float> staticBlockYPos = new List<float>()
+    {
+        5.5f, 5.5f, 5.5f, 5.5f, 5.5f, 5.5f, 5.5f,
+        7.5f, 7.5f, 7.5f, 7.5f, 7.5f, 7.5f, 7.5f,
+        9.5f, 9.5f, 9.5f, 9.5f, 9.5f, 9.5f, 9.5f,
+    };
+
+    public int numStaticBlocks = 21;
```

Comment:
If you say to create more than 21 static blocks, the game will crash, might want to use a range on this field.

Reply:
Good catch - set up a range of 1,21